async-stream-impl/src/lib.rs

```diff
@@ -225,8 +234,10 @@ pub fn stream_inner(input: TokenStream) -> TokenStream {
     quote!({
         let (mut __yield_tx, __yield_rx) = unsafe { #crate_path::__private::yielder::pair() };
         #crate_path::__private::AsyncStream::new(__yield_rx, async move {
-            #dummy_yield
-            #(#stmts)*
+            'check_stream_scope: {
```

Comment:
The label-break-value was stabilized in Rust 1.65 (https://github.com/rust-lang/rust/issues/48594), so this will increase the MSRV. This is okay since it is for soundness.

Could you update the documentation and rust-version fields?

https://github.com/tokio-rs/async-stream/blob/0588ec1e961e033ac0a84d30e50c86bf1029ddbe/async-stream/README.md?plain=1#L155
https://github.com/tokio-rs/async-stream/blob/0588ec1e961e033ac0a84d30e50c86bf1029ddbe/async-stream/Cargo.toml#L8
https://github.com/tokio-rs/async-stream/blob/0588ec1e961e033ac0a84d30e50c86bf1029ddbe/async-stream-impl/Cargo.toml#L5